test-requirements.txt

```diff
@@ -1,7 +1,4 @@
-flake8
```

Comment:
Don't we still use flake8?

Reply:
yes, but via pre-commit.  pre-commit manages the installation in a separate environment so you don't have to worry about system installation of tools

Reply:
I only like people to have the ability to run things manually.
- Sorry about all the questions, I am a GitHub noob, been locked away in Facebook for to long.

I'm not quite Dave Harrington n00b, but close.

Reply:
oh hey there @DaveHarrington -- small world I guess?